openslides/core/views.py

```diff
@@ -315,6 +329,42 @@ def clear_elements(self, request, pk):
         return Response(serializer.data)
 
     @detail_route(methods=['post'])
+    def set_resolution(self, request, pk):
+        """
+        REST API operation to set the resolution.
+
+        It is actually unused, because the resolution is currently set in the config.
+        But with the multiprojector feature this will become importent to set the
+        resolution per projector individually.
+
+        It expects a POST request to
+        /rest/core/projector/<pk>/set_resolution/ with a dictionary with the width
+        and height and the values.
+
+        Example:
+
+        {
+            "width": "1024",
+            "height": "768"
+        }
+        """
+        if not isinstance(request.data, dict):
+            raise ValidationError({'detail': 'Data must be a dictionary.'})
+        if (request.data['width'] <= 0 or
```

Comment:
Here you have to validate that someone does not send something like `{"foo": "bar"}`. At the moment you would get a KeyError.